src/Events/Appointments/AppoinmentSetEvent.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ *
+ * @package      OpenEMR
+ * @link               https://www.open-emr.org
+ *
+ * @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2021 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ */
+
+namespace OpenEMR\Events\Appointments;
+
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Event object for know what type of appointment has been set on the calendar
+ *
+ * @package OpenEMR\Events
+ * @subpackage Appointments
+ *
+ */
+class AppoinmentSetEvent extends Event
+{
+    /**
+     * This event is triggered after a new patient appointment has been scheduled
+     */
+    const EVENT_HANDLE = 'appointment.set';
+
+    /**
+     * @var
+     */
+    public $post;
```

Comment:
i think this can be private

Reply:
I always prefer class member variables to be private and to expose getter/setter variables if you need to get access to the data.  We are inconsistent throughout the codebase but its just good coding practice to not expose member variables except for strict POPO (Plain old PHP objects) which I don't think this falls under.